mutiny-core/src/nostr/mod.rs

```diff
@@ -102,6 +108,25 @@ pub struct NostrManager<S: MutinyStorage> {
     pub primal_client: PrimalClient,
 }
 
+/// A fedimint we discovered on nostr
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct NostrDiscoveredFedimint {
```

Comment:
should this be in the nostr rust library?

Reply:
Maybe eventually, when the spec is merged